src/Pim/Component/Catalog/spec/Normalizer/Indexing/Product/PropertiesNormalizerSpec.php

```diff
@@ -69,6 +71,13 @@ function it_normalizes_product_fields_and_values(
             ->willReturn($productValueCollection);
         $productValueCollection->isEmpty()->willReturn(false);
 
+        $product->getCategoryCodes()->willReturn(
```

Comment:
why is this method commented? If there is a reason, you should add a comment at the beginning.